Certification standard/main.html

```diff
@@ -0,0 +1,221 @@
+
+    <section id='ccwg-conformance'>
+      <h2> Certification process</h2> 
+
+      <section>
+        <h3> What is certified? </h3>
+
+        A certification refers to a DApp, which means the source code of the on-chain component of a smart contract or several smart contracts, along the UPLC compiled from the source code and a given parametrisation.
+
+        [Discussion: Do we want to certify Dapps with general parameters or only instantions of such Dapps?]
+      </section>
+      <section>
+        <h3> On-chain certificates </h3>
+        
+        Say something about our CIP on certification metadata.
+      </section>
+      <section>
+        <h3>Who can issue certificates ? </h3>
+
+        This section XXX
+        
+        [Todo:
+        - Do we keep a list of trusted issuer? Can anyone issue certificates?
+        - How do we join the list?
+        - How do we remove someone from the list?
+        ]
+      </section>
+    </section>
+
+    <section id='certlevels'>
+      <h2>Certification Levels</h2>
+
+      Three certification levels are defined, named 1, 2, and 3. [TBC: We have a chance to rename, redefine, reorganize, renumber the levels, etc.]
+
+      <section id='certlevels-l1'>
+        <h3>Level 1: Automated testing</h3> 
+
+        Certification at level 1 is intended for verification of a <a>DApp</a> that can be performed by an unguided automated tool. The tool shall be able to determine if the code meets its requirements and is not vulnerable to some vulnerabilities [Discussion: Should we relate to the list of vulnerabilities and define the ones that can/should be dealt at this level, because the tools can do it].
+
+        [Todo (or not but tbd):
+        - Vulnerabilities to be checked
+        - Metrics to be reached
+        - Tracability to documentation
+        - List of tools/techniques? Fuzzing, mutation testing, concolic testing, symbolic execution, property based testing, etc.?
+        - Tool qualification
+        - [Others?]
+        ]
+      </section>
+
+      <section id='certlevels-l2'>
+        <h3>Level 2: Manual audit</h3>
+
+        Certification at level 2 is intended for a <a>DApp</a> that has been carefully reviewed by an auditor, through a manual analysis with possibly the use of automated tools. [Discussion: How de we split this from auditor doing only the work of level 1?]
+
+        Say something about CIP 52.
+
+      </section>
+
+      <section id='certlevels-l3'>
+        <h3>Level 3: Formal verification</h3>
+
+        Certification at level 3 is intended for the verification of a <a>DApp</a> that went through formal verification.
+        
+        [Todo (or not but tbd):
+        - What needs to be proven?
+        - Should it really be its own level? 
+        - Who reviews the proof? The assumptions? The theorems?
+        - Tool qualification?
+        - Tracability paper spec/formalized spec/model
+        - [others]
+        ]
+      </section>
+
+
+      <section>
```

Comment:
It might be that some of these would be enforced by different levels: e.g. evidence for all level 2 audits of checking against a published vulnerability list. If that's the case then it might be that the remaining good practices should be published somewhere else (described here) rather than in the body of the standard. One justification for that would be that a change in the best practices would not require a formal change in the standard itself.